.github/workflows/builder.yml

```diff
@@ -32,12 +32,6 @@ on:
     branches:
       - main
       - "release-*"
-    paths-ignore:
```

Comment:
I have been executring the tests in my own [repo](https://github.com/phantomjinx/camel-k/actions)  but of course I `push` rather than create PRs. As a result, the `push` does not execute the e2e tests unless I remove this.

Reply:
Done